src/test/librados/c_write_operations.cc

```diff
@@ -259,15 +259,18 @@ TEST(LibRadosCWriteOps, CmpExt) {
   ASSERT_EQ(0, memcmp("five", hi, sizeof(hi)));
 
   // compare and bail before write due to mismatch
-  val = 0;
-  op = rados_create_write_op();
-  ASSERT_TRUE(op);
-  rados_write_op_cmpext(op, "four", 4, 0, &val);
-  rados_write_op_write(op, "six ", 4, 0);
-  ASSERT_EQ(-MAX_ERRNO - 1, rados_write_op_operate(op, ioctx, "test", NULL, 0));
-
-  ASSERT_EQ(-MAX_ERRNO - 1, val);
-
+  // do it 1000 times to make sure we are hitting
```

Comment:
_to make sure_ – a purist would argue it's about increasing the likelihood ;-).

Reply:
@rzarzynski indeed, we want to make sure we will hit on some point the socket failure injection and testing the error return value